app/models/notification.rb

```diff
@@ -426,8 +427,10 @@ def send_email
 #  index_notifications_on_topic_id_and_post_number              (topic_id,post_number)
 #  index_notifications_on_user_id_and_created_at                (user_id,created_at)
 #  index_notifications_on_user_id_and_topic_id_and_post_number  (user_id,topic_id,post_number)
-#  index_notifications_read_or_not_high_priority                (user_id,id DESC,read,topic_id) WHERE (read OR (high_priority = false))
-#  index_notifications_unique_unread_high_priority              (user_id,id) UNIQUE WHERE ((NOT read) AND (high_priority = true))
+#  index_notifications_read_or_not_high_priority                (user_id,old_id DESC,read,topic_id) WHERE (read OR (high_priority = false))
```

Comment:
This index should be dropped once we rename the old primary key to `old_id` as it will serve no purpose and take up disk space.

Reply:
I wanted to keep them in case we need to rollback to using `old_id`. In that case, we would not have to regenerate the indexes again.